web-extensions/chrome/scripts/load-bookmarks.js

```diff
@@ -270,20 +246,28 @@ BmcMangaList.prototype.filter = function(filterStr) {
     }
 };
 
+function showDeleteButton() {
+    displayButton(document.getElementById('delete-but'));
```

Comment:
There's a display but no hide function ?
Might be part of a rationalization effort.

Reply:
Will be handled through #53